dev/tasks/generate-crds

```diff
@@ -38,6 +38,7 @@ go run ./scripts/crd-tools set-field spec.preserveUnknownFields=false --dir apis
 go run ./scripts/crd-tools delete-field status --dir apis/config/crd/
 go run ./scripts/crd-tools set-annotation cnrm.cloud.google.com/version=0.0.0-dev --dir apis/config/crd/
 go run ./scripts/crd-tools delete-annotation controller-gen.kubebuilder.io/version --dir apis/config/crd/
+go run ./scripts/crd-tools reflow-descriptions --dir apis/config/crd/
```

Comment:
We don't have apis/config/crd dir yet. Is this for future structure?

Reply:
So actually this script generates that directory.  We don't check it in currently, and it does get "left behind" if you run it locally.  I'm not sure if we should delete it in the script (or generate it under `.build`).  WDYT?

Reply:
Oh I see what this script does. (I was more thinking about keeping config/apis as the single CRD dir)

I"ll vote for `.build`.